src/tests/operate.test.js

```diff
@@ -0,0 +1,55 @@
+// eslint-disable-next-line
+import operate from '../logic/operate';
```

Comment:
- [ ] I ran this test and I got this error seen in the image below. This is because the folder `Logic` is supposed to have an uppercase L at the beginning. It should be like this `../Logic/operate`
![image](https://user-images.githubusercontent.com/45460192/228631364-f226dfb6-fa6f-475a-83f0-246f6486ad7b.png)